sdk/core/core-http/tsconfig.json

```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "module": "commonjs",
     "outDir": "dist",
-    "target": "es5",
```

Comment:
I think we have to major bump core-http for this, since `ServiceClient` is extended by generated clients (which could be targeting ES5 still, like Storage)

Reply:
Oh yes that went over my head! I reverted the change there for now and I am not really sure if it is worth doing given that we are migrating everyone off core-http in the next semester.